boards/v2/raspberrypi/rpi_pico/Kconfig.rpi_pico_w

```diff
@@ -0,0 +1,5 @@
+# Copyright (c) 2021 Yonatan Schachter
```

Comment:
the `BOARD` is rpi_pico, and `w` is a variant, so this file will not be loaded and can be removed.
https://github.com/zephyrproject-rtos/zephyr/blob/3023de168e97eeb01cd4afbee5d71f8605e3bbea/boards/Kconfig.v2#L18

Reply:
Fixed